hw3/src/main.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
+import { createBrowserRouter, RouterProvider} from "react-router-dom"
+
+import Home from './routes/Home';
+import List from './routes/List';
+import Error from './routes/Error';
+
+const rootElement = document.getElementById('root');
+const root = createRoot(rootElement);
+
+//Build the router
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Home />,
+    errorElement: <Error />,
+  },
+  {
+    path: "/list",
+    element: <List />,
+  },
+]);
+
+root.render(
+  <StrictMode>
+    <RouterProvider router={router} />
+  </StrictMode>
+);
```

Comment:
I could see that you have not implemented the population yet, as you have mentioned that you were trying to solve the list Problem. Kindly check my comments of resolving the list problem.